structuredevent-model/src/main/java/com/sequenceiq/cloudbreak/structuredevent/event/CloudbreakEventService.java

```diff
@@ -9,6 +9,7 @@
 import com.sequenceiq.cloudbreak.event.ResourceEvent;
 
 public interface CloudbreakEventService {
+    // todo: make an enum instead of string constants
```

Comment:
Earlier we cannot make these strings as Enum because we don't have a common module if I remember correctly. If you can make strings as Enum and you can access these in every module, I'm happy with that

Reply:
I took a look at making this change, and it's going to be a lot of churn. I'll remove the todo and just leave things as they are for the moment.